content/manual-references.json

```diff
@@ -38,5 +38,24 @@
         "literal": "NVIDIA"
     }
   ]
+},
+{
+  "type": "webpage",
+  "standard_citation": "url:https://github.com/Benjamin-Lee/deep-rules",
+  "URL": "https://github.com/Benjamin-Lee/deep-rules",
+  "title": "Benjamin-Lee/deep-rules GitHub repository",
```

Comment:
I might add

```json
"container-title": "GitHub",
```

Reply:
I agreed with this suggestion and made the change